webapp/src/index.tsx

```diff
@@ -44,18 +45,30 @@ document.addEventListener('DOMContentLoaded', () => {
 
         root.render(
             <React.StrictMode>
-                {!validEnvFile && <MissingEnvVariablesError missingVariables={missingEnvVariables} />}
-                {validEnvFile && (
+                {validEnvFile ? (
                     <ReduxProvider store={store}>
-                        {/* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */}
-                        <MsalProvider instance={msalInstance!}>
-                            <FluentProvider className="app-container" theme={semanticKernelLightTheme}>
-                                <App />
-                            </FluentProvider>
-                        </MsalProvider>
+                        {/* eslint-disable @typescript-eslint/no-non-null-assertion */}
+                        {shouldUseMsal ? (
+                            <MsalProvider instance={msalInstance!}>
+                                <AppWithTheme />
+                            </MsalProvider>
+                        ) : (
+                            <AppWithTheme />
+                        )}
+                        {/* eslint-enable @typescript-eslint/no-non-null-assertion */}
```

Comment:
this is needed, notice that it's an `eslint-enable` comment instead of a disable. it's because `eslint-disable-next-line` cannot be used above since the comment is not allowed inside the ternary.